packages/core/utils/src/config.js

```diff
@@ -76,6 +82,11 @@ export async function loadConfig(
 ): Promise<ConfigOutput | null> {
   let configFile = await resolveConfig(fs, filepath, filenames, opts);
   if (configFile) {
+    let cachedOutput = configCache.get(configFile);
+    if (cachedOutput) {
+      return cachedOutput;
+    }
+
     try {
       let extname = path.extname(configFile).slice(1);
       if (extname === 'js') {
```

Comment:
Should we set the cache here too before returning?

```
configCache.set(configFile, {
  // $FlowFixMe
  config: clone(require(configFile)),
  files: [{filePath: configFile}],
});
```

Reply:
good catch!